build/SignAssembly.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+
+  <PropertyGroup>
+    <SignAssembly>True</SignAssembly>
+    <AssemblyOriginatorKeyFile>$(RootDirectory)Moq.snk</AssemblyOriginatorKeyFile>
+    <PublicSign Condition="'$(OS)' != 'Windows_NT'">True</PublicSign>
```

Comment:
Can't we public sign on Windows? Really?

Reply:
No idea, didn't check that one yet. But it's been there before, it's no new addition.

Reply:
@kzu, after reading https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/public-signing.md:

> To enable you to build assemblies that have a matching identity to what Microsoft would build, we leverage a new signing mechanism called *Public Signing*.

Why would we even want to public-sign at all? Moq's keypair used for strong-naming is available to everyone as it's included in the repository, so public-signing simply shouldn't be necessary.

It seems to me that the `<PublicSign>True</PublicSign>` should be removed.